### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by nmarko91** [claimsforce-gmbh/flutter-segment#96]
After updating to `3.4.0` to be able to use Dart config instead of Native config (as seen here https://github.com/claimsforce-gmbh/flutter-segment/issues/87), when I run pod install I get the error:

```
[!] The 'Pods-Runner' target has transitive dependencies that include statically linked binaries: (Segment-Amplitude)
```

Any ideas on how to fix this one? 

Thanks

**Comment by jeremynorman89** [comment]
I'm also experiencing this!

**Comment by MaiKaY** [comment]
@danielgomezrico @esarbanis @ariefwijaya do you think this will be solved with this PR https://github.com/claimsforce-gmbh/flutter-segment/pull/97?

**Comment by esarbanis** [comment]
@MaiKaY I think so, yes.

**Comment by MaiKaY** [comment]
Should be fixed in `3.4.1`
